### PR TITLE
Release creation like CaCo3's update-ota2

### PR DESCRIPTION
**Pull request by haverland**
- update.zip now like update-rolling
- used initial-setup.zip instead of standalone files
- initial-setup.zip contains now the complete sd-card directory
- updated the Changelog.md in Unreleased part
- suppress extracting of directories in OTA  update

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -158,6 +158,18 @@ jobs:
         name: "update__${{ github.ref_name }}_(${{ steps.vars.outputs.sha_short }})"
         path: ./dist/*
 
+    - name: Store generated files in cache
```
Comment (caco3) [inline .github/workflows/build.yaml:161]: Unless you modified the files, there is no need to push them to (another) cache!

File: .github/workflows/build.yaml [hunk 2]
```diff
@@ -158,6 +158,18 @@ jobs:
...
+          ./code/.pio/build/esp32cam/partitions.bin
+          ./code/.pio/build/esp32cam/bootloader.bin
+          ./sd-card/html/version.txt
+          ./dist
+        key: ${{ github.run_number }}-pack-for-OTA-v2
+
+
 
 
 #########################################################################################
```
Comment (caco3) [inline .github/workflows/build.yaml:175]: `pack-for-fresh-install` doesn't have any use anymore, right?

File: code/components/jomjol_fileserver_ota/server_file.cpp [hunk 3]
```diff
@@ -49,8 +49,8 @@ extern "C" {
 
 /* Max size of an individual file. Make sure this
  * value is same as that set in upload_script.html */
-#define MAX_FILE_SIZE   (2000*1024) // 200 KB
-#define MAX_FILE_SIZE_STR "2000KB"
+#define MAX_FILE_SIZE   (8000*1024) // 8 MB
```
Comment (caco3) [inline code/components/jomjol_fileserver_ota/server_file.cpp:52]: I would move this in a separate PR!
maybe combine it with https://github.com/jomjol/AI-on-the-edge-device/pull/1073?

File: .github/workflows/build.yaml (hunk 2 above)
Comment (haverland) [reply .github/workflows/build.yaml:175]: I did't saw it. Will change the build structure for cleaner reading.
